README.md

```diff
@@ -1 +1,190 @@
-# Microverse-hello-project
\ No newline at end of file
+# Microverse-hello-project
+# 
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+- [👥 Authors](#authors)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+  -- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ](#faq)
+- [📝 License](#license)
+
+
+# 📖 [Microverse-hello-project] <a name="about-project"></a>
+
+**[Microverse-hello-project]** Practicalizing how to use Linters in a project and following github workflow.
+
+## 🛠 Built With <a name="built-with"></a>
+html and css
+
+### Tech Stack <a name="tech-stack"></a>
+
+  <summary>Javascript runtime environment</summary>
+  <ul>
+    <li><a href="https://nodejs.org/en/">Node JS</a></li>
+  </ul>
+
+  <summary>Version control</summary>
+  <ul>
+    <li><a href="github.com">Git Hub</a></li>
+  </ul>
+</details
```

Comment:
- [ ] Instead of NodeJS, GitHub, Version control you may consider saying HTML and CSS since they are the tech stack you used in this project. you may consider adding these technologies you put here in the prerequisite.